logo/print_logo.ts

```diff
@@ -0,0 +1,8 @@
+import { dirname, fromFileUrl } from "std/path";
+
+export async function printLogo(): Promise<void> {
+  await Deno.run({
+    cmd: ["./logo.sh"],
+    cwd: dirname(fromFileUrl(import.meta.url)),
```

Comment:
One question about `import.meta.url`. What is this?

Reply:
`import.meta.url` gets the path of the current file. I'm stripping the `file:///` in its output with `fromFileUrl`, and using only the directory path with `dirname`. `cwd` in `Deno.run` guarantees that this command will be run in that directory, in this case, `/logo` inside the project path.